pandas/tools/pivot.py

```diff
@@ -382,7 +382,9 @@ def crosstab(index, columns, values=None, rownames=None, colnames=None,
     columns : array-like, Series, or list of arrays/Series
         Values to group by in the columns
     values : array-like, optional
-        Array of values to aggregate according to the factors
+        Array of values to aggregate according to the factors.
```

Comment:
As we discussed last week, `normalize` doesn't mix well with an `aggfunc`, but this is a clear situation where we'd want to be able to normalize values (as per @sinhrks suggestion). This makes that possible.